.github/workflows/haros.yml

```diff
@@ -0,0 +1,58 @@
+name: Haros Analysis
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
```

Comment:
Should we add on pull request on master/push (even if they will not be that common)? I mean we don't even pay for the runners anyway.

A bit out of scope, why do we have develop and master?

Reply:
Having `develop` is simply an old habit of mine. It feels easier having a develop branch to QA the integration of all the features before a release. But I wouldn't mind changing it.

I'll add triggers for branch `master`.